molgenis-core-ui/src/main/javascript/modules/react-components/Form.js

```diff
@@ -272,9 +285,10 @@
 					)
 				)
 			) : null;
-					
+
+			//TODO: Rename; this shadows the Form class!
```

Comment:
I'm worried that rewriting it causes new bugs, we're doing this with our refactor hat on: rewrite as little as possible, leave everything working the same.